godot-core/src/obj/gd.rs

```diff
@@ -207,6 +207,10 @@ where
 
 /// _The methods in this impl block are available for any `T`._ <br><br>
 impl<T: GodotClass> Gd<T> {
+    pub(crate) fn drop_weak(self) {
+        std::mem::forget(self);
+    }
+
```

Comment:
Please move it next to other `pub(crate)` fn and add a bit of documentation:
```suggestion
    /// Drop without decrementing ref-counter.
    ///
    /// Needed in situations where the instance should effectively be forgotten, but without leaking other associated data.
    pub(crate) fn drop_weak(self) {
        // As soon as fields need custom Drop, this won't be enough anymore.
        std::mem::forget(self);
    }

```